project/src/main/java/com/google/sps/data/UserPrefrences.java

```diff
@@ -0,0 +1,109 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import com.google.auto.value.AutoValue;
+import com.google.maps.model.LatLng;
+
```

Comment:
There should be no space between these imports from what I understand (Ariel told me the favourable way to do this is by having a block of all static imports, then one new line, and then all the non-static imports).

Reply:
Right, I also saw it on the java style guide 👍